packages/react-native/src/Authenticator/common/DefaultContent/styles.ts

```diff
@@ -0,0 +1,33 @@
+import { StyleSheet } from 'react-native';
+
+import { StrictTheme } from '../../../theme';
+import { DefaultContentStyle } from './types';
+
+export const getDefaultStyle = ({
+  tokens: { space },
+}: StrictTheme): Required<DefaultContentStyle> =>
+  StyleSheet.create({
+    buttonPrimary: {
+      marginVertical: space.medium,
+    },
+    buttonPrimaryLabel: {}, // themed value only
```

Comment:
These empty objects are temporary until we have added subcomponent theming support